### PR TITLE
App strings reworked 2

### PR DESCRIPTION
**Pull request by comradekingu**
What does
```
    <string name="notification_visibility_public">Publicly visible notification</string>
    <string name="notification_visibility_private">Privately visible notification</string>
```
mean?

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -27,12 +27,12 @@
...
+    <string name="locks">Lockings</string>
+    <string name="unlocks">Unlockings</string>
     <string name="content_description_lock_screen_filter">Lockscreen filter</string>
-    <string name="locks_and_unlocks">Locks and unlocks</string>
-    <string name="locks_only">Locks only</string>
-    <string name="unlocks_only">Unlocks only</string>
+    <string name="locks_and_unlocks">Lockings and unlockings</string>
+    <string name="locks_only">Lockingss only</string>
+    <string name="unlocks_only">Unlockings only</string>
     <string name="disconnected">Disconnected</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:36]: Does this mean "unplugged"?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -58,34 +58,34 @@
...
     <string name="power_disconnected">Power disconnected</string>
     <string name="power_connected">Power connected</string>
     <string name="ac_charging">AC charging</string>
     <string name="usb_charging">USB charging</string>
     <string name="wireless_charging">Wireless charging</string>
     <string name="error_occurred">An error occurred. Try again later.</string>
     <string name="channel_name">Vigilante usage notifications</string>
-    <string name="channel_description">Notifications are sent when privacy usage happens</string>
-    <string name="mic_being_used">Your microphone is being used!</string>
+    <string name="channel_description">Notifications are sent when usage occurs</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:71]:
```suggestion
    <string name="channel_description">Notifications are sent when privacy usage occurs</string>
```
I don't really understand what this word does here, so I am not sure it isn't needed.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -43,11 +43,11 @@
     <string name="always_allowed">Allowed always</string>
     <string name="allowed_one_time">Allowed only once</string>
     <string name="denied">Denied</string>
-    <string name="denied_and_do_not_ask_again">"Denied permanently "</string>
+    <string name="denied_and_do_not_ask_again">"Denied permanently"</string>
     <string name="ask_every_time">Ask each time</string>
     <string name="permissions_requests_by_app_count">%1$d permission requests in total.</string>
     <string name="total_permission_requests">Total permission requests</string>
-    <string name="title_placeholder">Title: %1$s</string>
+    <string name="title_placeholder">Name: %1$s</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:50]: ?

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -181,10 +181,10 @@
     <string name="english">English</string>
     <string name="french">French</string>
     <string name="german">German</string>
-    <string name="italian">Italian</string>
-    <string name="marathi">Marathi</string>
     <string name="hebrew">Hebrew</string>
+    <string name="italian">Italian</string>
     <string name="macedonian">Macedonian</string>
+    <string name="marathi">Marathi</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:187]: No Norwegian Bokmål?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (FunkyMuse) [reply app/src/main/res/values/strings.xml:36]: no

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (FunkyMuse) [reply app/src/main/res/values/strings.xml:71]: when one of the things vigilante notifies you about is triggered

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:36]:
```suggestion
    <string name="disconnected">Unplugged</string>
```

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -27,12 +27,12 @@
...
+    <string name="unlocks">Unlockings</string>
     <string name="content_description_lock_screen_filter">Lockscreen filter</string>
-    <string name="locks_and_unlocks">Locks and unlocks</string>
-    <string name="locks_only">Locks only</string>
-    <string name="unlocks_only">Unlocks only</string>
+    <string name="locks_and_unlocks">Lockings and unlockings</string>
+    <string name="locks_only">Lockingss only</string>
+    <string name="unlocks_only">Unlockings only</string>
     <string name="disconnected">Disconnected</string>
     <string name="connected">Connected</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:37]:
```suggestion
    <string name="connected">Plugged in</string>
```

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -138,28 +138,28 @@
     <string name="notification_led_color">Notification LED color</string>
     <string name="pick_notification_LED_color">Pick a notification LED-color</string>
     <string name="led_notification_expl">Only available for devices with a notification LED.</string>
-    <string name="bypass_dnd_title">Bypass Do Not Disturb</string>
-    <string name="bypass_dnd_expl">Notifications about usage in all modes. Use with caution!</string>
+    <string name="bypass_dnd_title">Disregard \"Do not disturb\"</string>
+    <string name="bypass_dnd_expl">Notifications of usage in all modes. Use with caution!</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:142]:
```suggestion
    <string name="bypass_dnd_expl">Notifications of usage in all modes.\nUse with caution!</string>
```

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:71]: All other occurrences are just "usage", so going with the consistent option.

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -1,22 +1,22 @@
 <resources>
     <string name="app_name" translatable="false">Vigilante</string>
     <string name="vigilante_description">Vigilante service</string>
-    <string name="vigilante_summary">Vigilante service listener for events that displays dot when camera/mic usage happens.</string>
+    <string name="vigilante_summary">Listens for events and displays a dot when camera/mic usage occurs.</string>
```
Comment (FunkyMuse) [inline app/src/main/res/values/strings.xml:4]: Vigilante Listens

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (FunkyMuse) [reply app/src/main/res/values/strings.xml:50]: should be title

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (FunkyMuse) [reply app/src/main/res/values/strings.xml:187]: had it forgotten tbh, it doesn't exist in settings preferences

File: app/src/main/res/values/strings.xml (hunk 7 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml:4]: Thought it went without saying with the title, but forgot where it is found.

Comment (comradekingu) [inline app/src/main/res/values/strings.xml:4]:
```suggestion
    <string name="vigilante_summary">Vigilante listens for events and displays a dot when camera/mic usage occurs.</string>
```

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:50]:
```suggestion
    <string name="title_placeholder">Title: %1$s</string>
```

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -138,28 +138,28 @@
...
+    <string name="history_deletion">Limit history</string>
+    <string name="history_deletion_expl">Only keep the last 30 days</string>
     <string name="application">App</string>
     <string name="dot_indicator_customizations">Dot-indicator customizations.</string>
     <string name="generate">Generate</string>
     <string name="generate_one_time_password">Generate one-time password</string>
-    <string name="password_length">8 characters or more</string>
-    <string name="password_length_expl">Fewer characters makes your password easier to guess</string>
+    <string name="password_length">8+ characters</string>
+    <string name="password_length_expl">(Fewer characters makes your password easier to guess</string>
```
Comment (FunkyMuse) [inline app/src/main/res/values/strings.xml:162]: Shouldn't end with ) as well?

Comment (comradekingu) [inline app/src/main/res/values/strings.xml:162]:
```suggestion
    <string name="password_length_expl">(Fewer characters makes your password easier to guess)</string>
```

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -58,34 +58,34 @@
...
     <string name="error_occurred">An error occurred. Try again later.</string>
     <string name="channel_name">Vigilante usage notifications</string>
-    <string name="channel_description">Notifications are sent when privacy usage happens</string>
-    <string name="mic_being_used">Your microphone is being used!</string>
+    <string name="channel_description">Notifications are sent when usage occurs</string>
+    <string name="mic_being_used">Your mic is being used!</string>
     <string name="location_being_used">Your location is turned on, could be it\'s being used!</string>
     <string name="cam_being_used">Your camera is being used!</string>
     <string name="usage_title">Usage detected</string>
     <string name="dot_usage_title">Show dot usage</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:76]:
```suggestion
    <string name="dot_usage_title">Show usage dot</string>
```

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -58,34 +58,34 @@
...
-    <string name="dot_usage_expl">Shows a dot appear on your screen during use.</string>
+    <string name="dot_usage_expl">Displays a dot on your screen during use.</string>
```
Comment (comradekingu) [inline app/src/main/res/values/strings.xml:77]:
```suggestion
    <string name="dot_usage_expl">Displays a dot on your screen during specified use.</string>
```

**Review by FunkyMuse — APPROVED** [review]

**Pull request merged**